aqw-opcua-server/Settings.h

```diff
@@ -1,24 +1,24 @@
 #pragma once
-#include "cpprest/http_client.h"
+
 #include <fstream>
 #include <iostream>
 
-namespace weathersvr {
+#include <cpprest/http_client.h>
+
+namespace weatherserver {
 
     class Settings {
 
         public:
 
             Settings();
-            /*
-            Open the file passed to the formal parameter fileName and set the settings to variables of this class.
-            @param char* fileName - the path containing the settings file name and extension that is passed to command line arguments.
-            */
-            bool setup(char* fileName);
+
+            //Open the SETTINGS_FILE_NAME file and assign values to variables.
+            bool setup();
 
             const utility::string_t& getKeyApiDarksky() const { return keyApiDarksky; }
             const utility::string_t& getUnits() const { return units; }
-            const short getIntervalDownloadWeatherData() const { return intervalDownloadWeatherData; }
+            const int getIntervalWeatherDataDownload() const { return intervalWeatherDataDownload; }
```

Comment:
No need to return `const int`, just return `int` because it will a copy and client code can do whatever it wants.